packages/api-reference/vite.standalone.config.ts

```diff
@@ -19,6 +18,12 @@ export default defineConfig({
   define: {
     'process.env.NODE_ENV': '"production"',
   },
+  resolve: {
+    alias: {
+      '@': fileURLToPath(new URL('./src', import.meta.url)),
```

Comment:
<img src="https://media3.giphy.com/media/5VKbvrjxpVJCM/giphy.gif"/>
we get aliases in references now?!?!

Reply:
Haha we have had them for ages.